### PR TITLE
Use Mandrill to send e-mail

### PR DESCRIPTION
**Pull request by JustynaBroniszewska**

### CODE REVIEW

File: universal-login-relayer/src/config/baseConfig.ts [hunk 1]
```diff
@@ -33,4 +33,14 @@ export const baseConfig = {
   emailPassword: getEnv('EMAIL_PASSWORD', ''),
   copyToClipboardUrl: 'https://universal-provider-backend.netlify.app/copy',
   emailLogo: UNILOGIN_LOGO_WITH_NAME_URL,
+  mailingCredentials: {
+    transport: {
```
Comment (xitronix) [inline universal-login-relayer/src/config/baseConfig.ts:37]: we can remove redundant layer `transport`

File: universal-login-relayer/src/config/baseConfig.ts [hunk 2]
```diff
@@ -33,4 +33,14 @@ export const baseConfig = {
   emailPassword: getEnv('EMAIL_PASSWORD', ''),
   copyToClipboardUrl: 'https://universal-provider-backend.netlify.app/copy',
   emailLogo: UNILOGIN_LOGO_WITH_NAME_URL,
+  mailingCredentials: {
```
Comment (xitronix) [inline universal-login-relayer/src/config/baseConfig.ts:36]: we can change mailingCredentials to `email` and put into this variable all email props:
```ts
{
  emailPassword: ..
  emailLogo:
   copyToClipboardUrl
...
```

File: universal-login-relayer/src/config/config.ts [hunk 3]
```diff
@@ -29,6 +25,14 @@ export interface Config {
   maxGasLimit: number;
   ipGeolocationApi: IPGeolocationApiConfig;
   httpsRedirect: boolean;
+  email: Email;
+}
+
+export interface Email {
```
Comment (xitronix) [inline universal-login-relayer/src/config/config.ts:31]: What do you think about to name it`EmailConfig`

Comment (JustynaBroniszewska) [reply universal-login-relayer/src/config/config.ts:31]: ok

File: universal-login-relayer/tsconfig.json [hunk 4]
```diff
@@ -3,10 +3,11 @@
...
     "allowJs": true,
     "jsx": "react",
-    "outDir": "dist"
+    "outDir": "dist",
+    "typeRoots": ["shims"]
   },
   "include": [
     "src",
-    "test"
+    "test",
```
Comment (JustynaBroniszewska) [inline universal-login-relayer/tsconfig.json:11]:
```suggestion
    "test"
```

**Pull request merged**